crates/abi/src/bindings/alloy_console.rs

```diff
@@ -0,0 +1,28 @@
+//! Alloy console bindings.
```

Comment:
What is this for? I wouldn't do this to avoid conflicts with #5998

Reply:
This is for migrating the console bindings so decoding utilities can be migrated—but happy to do it after #5998, as it's closely related to cheatcodes